src/commands/new.ts

```diff
@@ -195,8 +195,12 @@ export default {
     // #endregion
 
     // #region Project Path
+    const parsePath = (p: string) =>
+      p.startsWith("~") ? path(p.replace("~", filesystem.homedir())) : path(p)
     const defaultTargetPath = path(projectName)
-    let targetPath = useDefault(options.targetPath) ? defaultTargetPath : options.targetPath
+    let targetPath = useDefault(options.targetPath)
+      ? defaultTargetPath
+      : parsePath(options.targetPath)
```

Comment:
Wrapping flag args in `filesystem.path` expands `./Code/PizzaApp` to `~/Code/PizzaApp`